iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -437,7 +436,7 @@ public ShellForm SHELL_FORM() {
     return b.<ShellForm>nonterminal(DockerLexicalGrammar.SHELL_FORM).is(
       f.shellForm(
         b.oneOrMore(
-          b.token(STRING_LITERAL)
+          f.ignoreFirst(b.token(DockerLexicalGrammar.WHITESPACE), ARGUMENT_GENERIC())
```

Comment:
I ignore every whitespace.
Maybe instead we should only ignore the first one, and then capture other whitespace and make them expression/argument, as it is done for KeyValuePair ?

Reply:
Since they will result in spaces in the resulting command, we should deal with them.